### PR TITLE
Add the Handle Survey Results help topics

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: docs/handle-survey-results-access.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+console.log(npsQuestion.value);
+// ===== or =====
+// console.log(survey.getValue("nps_score"));
+
+npsQuestion.value = 10;
+// ===== or =====
+// console.log(survey.setValue("nps_score", 10));
+```
+
+If you need to access an individual question value before or right after it is changed, handle the Survey's [`onValueChanging`](/Documentation/Library?id=surveymodel#onValueChanging) or [`onValueChanged`](/Documentation/Library?id=surveymodel#onValueChanged) events:
```
Comment (dmitry-eliseev-devexpress) [inline docs/handle-survey-results-access.md:38]:
```suggestion
If you need to access an individual question value before or immediately after it is changed, handle the Survey's [`onValueChanging`](/Documentation/Library?id=surveymodel#onValueChanging) or [`onValueChanged`](/Documentation/Library?id=surveymodel#onValueChanged) events:
```

File: docs/handle-survey-results-access.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+```js
+{
+  "question1Name": "question1Value",
+  "question2Name": "question2Value",
+  // ...
+  "questionNName": "questionNValue",
+}
+```
+
+Depending on the question type, question values can also be nested objects or arrays that contain objects or primitive values:
```
Comment (dmitry-eliseev-devexpress) [inline docs/handle-survey-results-access.md:65]: What do you mean by "primitive values"? Simple values compared to objects?

```suggestion
Depending on the question type, question values can also be nested objects or arrays that contain objects or simple values:
```

File: docs/handle-survey-results-access.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+});
+
+survey.onValueChanged.add((survey, { name, question, value }) => {
+  console.log(`The ${name} question value has changed to ${value}.`);
+});
+```
+
+## Access Full Survey Results
+
+To access full survey results, use the Survey's [`data`](/Documentation/Library?id=surveymodel#data) property. It is a JSON object that contains only answered questions and has the following structure:
```
Comment (dmitry-eliseev-devexpress) [inline docs/handle-survey-results-access.md:54]: > property. **It** is a JSON object that contains only answered questions

Did you want to say that "The property value is a JSON object ..."?

Can a property be an object itself?

File: docs/handle-survey-results-continue-incomplete.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# Continue an Incomplete Survey
+
+Your respondents may not complete your survey in a single session. In this case, you can restore their answers from the previous session the next time they get to the survey. Incomplete results can be loaded from your database or the browser's [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage).
```
Comment (dmitry-eliseev-devexpress) [inline docs/handle-survey-results-continue-incomplete.md:3]:
```suggestion
Your respondents may not complete your survey in a single session. In this case, you can restore their answers from the previous session next time they get to the survey. Incomplete results can be loaded from your database or the browser's [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage).
```

File: docs/handle-survey-results-store.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
+# Store Survey Results
+
+Survey results are JSON objects that you can store in your own database or in the SurveyJS Service.
+
+- [Store Survey Results in Your Own Database](#store-survey-results-in-your-own-database)
+- [Store Survey Results in the SurveyJS Service](#store-survey-results-in-the-surveyjs-service)
+
+## Store Survey Results in Your Own Database
+
+To store survey results in your own database, handle the [onComplete](https://surveyjs.io/Documentation/Library?id=surveymodel#onComplete) event. It is raised when a user clicks the Complete button. The following code shows how to use this event to send survey results to your server:
```
Comment (dmitry-eliseev-devexpress) [inline docs/handle-survey-results-store.md:10]:
```suggestion
To store survey results in your own database, handle the [onComplete](https://surveyjs.io/Documentation/Library?id=surveymodel#onComplete) event raised in response to a click on the Complete button. The following code shows how to use this event to send survey results to your server:
```

File: docs/handle-survey-results-store.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+    } else {
+      // Display the "Error" message (pass a string value to display a custom message)
+      options.showDataSavingError();
+    }
+  };
+  xhr.send(JSON.stringify(sender.data));
+});
+```
+
+The `onComplete` event handler only sends survey results in JSON format to your server. How you store them fully depends on your backend.
```
Comment (dmitry-eliseev-devexpress) [inline docs/handle-survey-results-store.md:39]:
```suggestion
The `onComplete` event handler only sends survey results in JSON format to your server. The way you store them fully depends on your backend.
```

**Review by dmitry-eliseev-devexpress — APPROVED** [review]

File: docs/handle-survey-results-access.md (hunk 2 above)
Comment (RomanTsukanov) [reply docs/handle-survey-results-access.md:65]: Primitive [is a proper term](https://developer.mozilla.org/en-US/docs/Glossary/Primitive) in JS.

**Review by annnke — APPROVED** [review]
good good )

**Pull request merged**